tasks/pip.yml

```diff
@@ -124,14 +122,13 @@
   notify: reload supervisor
 
 - name: Install custom OpenWISP 2 Python packages
-  when: openwisp2_python_packages
+  when: openwisp2_python_packages|length > 0
   pip:
     name: "{{ openwisp2_python_packages }}"
-    state: latest
+    state: forcereinstall
     virtualenv: "{{ virtualenv_path }}"
     virtualenv_python: "{{ openwisp2_python }}"
     virtualenv_site_packages: yes
-    extra_args: "--no-deps"
```

Comment:
when using custom targets (eg: the master branches) I found that the system would download and install the dependencies from pypi, overriding the custom dependencies just downloaded.

Eg:

- openwisp-utils 0.6.0 alpha is installed from github
- installing openwisp-controller master from github downloads and installs openwisp-utils 0.5.0 stable, hence losing openwisp-utils 0.6.0. I haven't found a solution to this yet, but if you do (it's quite likely it can be done since I didn't have a lot of time to dedicate to this) please let me know and maybe we can solve another painful point here

Reply:
I am not able to replicate it,
I did:
```
openwisp2_utils_pip: https://github.com/atb00ker/openwisp-utils/tarball/master
openwisp2_controller_pip: https://github.com/openwisp/openwisp-controller/tarball/master
```

Where I randomly put util's version as `0.9.0` in my fork's master branch and I am seeing `0.9.0` only.

Reply:
ok, let's remove it, I'll try an alternative and cleaner solution